Makefile

```diff
@@ -2,51 +2,69 @@ CC ?= gcc
 CFLAGS += -std=c99 -Wall -O3
 LDFLAGS += -lm
 MAKE ?= make
+AUTORECONF ?= autoreconf
+CMAKE ?= cmake
 PREFIX ?= /usr/local
 
 UNAME_S := $(shell uname -s)
-
 ifeq ($(UNAME_S),Linux)
-	# Linux (e.g. Ubuntu)
-	MOZJPEG_PREFIX ?= /opt/mozjpeg
-	CFLAGS += -I$(MOZJPEG_PREFIX)/include
-	LIBJPEG = $(MOZJPEG_PREFIX)/lib/libjpeg.a
+	OS = Unixy
 else ifeq ($(UNAME_S),Darwin)
-	# Mac OS X
-	MOZJPEG_PREFIX ?= /usr/local/opt/mozjpeg
-	LIBJPEG = $(MOZJPEG_PREFIX)/lib/libjpeg.a
-	CFLAGS += -I$(MOZJPEG_PREFIX)/include
+	OS = Unixy
 else ifeq ($(UNAME_S),FreeBSD)
-	# FreeBSD
-	LIBJPEG = $(PREFIX)/lib/mozjpeg/libjpeg.so
-	CFLAGS += -I$(PREFIX)/include/mozjpeg
+	OS = Unixy
+else
+	OS = Windows
+endif
+
+ifeq ($(OS),Unixy)
+	LIBJPEG = src/mozjpeg/.libs/libjpeg.a
 else
-	# Windows
-	LIBJPEG = ../mozjpeg/libjpeg.a
-	CFLAGS += -I../mozjpeg
+	LIBJPEG = src/mozjpeg/libjpeg.a
 endif
+CFLAGS += -Isrc/mozjpeg
 
 LIBIQA=src/iqa/build/release/libiqa.a
 
 all: jpeg-recompress jpeg-compare jpeg-hash
 
+src/mozjpeg:
+	git clone -b v3.3.1 --single-branch https://github.com/mozilla/mozjpeg.git $@
+
+$(LIBJPEG): src/mozjpeg
+ifeq ($(OS),Unixy)
+	cd $< && \
+		$(AUTORECONF) -fiv && \
```

Comment:
This will no longer work :(
You need to use **cmake** to build _mozjpeg_ now.
Something similar to this
`mkdir build && cd build && cmake -G"Unix Makefiles" ../  -DCMAKE_INSTALL_PREFIX=/usr/local -DWITH_JPEG8=1 && make && make install/strip && ldconfig -n /usr/local/lib`